lib/factory_girl/strategy/stub.rb

```diff
@@ -68,6 +69,19 @@ def created_at
           end
         end
       end
+
+      def clear_changed_attributes_on_result(result_instance)
+        unless result_instance.respond_to?(:clear_changes_information)
```

Comment:
could we denote this as a "more private" method by using the double-under prefix/suffix? e.g. `:__clear_changes_information__`